decidim/decidim
# Allow deletion of categories when there are no resources associated

#### :tophat: What? Why?
This PR adds `dependent: :destroy` to `:categorization` to so that the link between resource and category to be removed. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #8389

#### Testing
1. Login as admin 
2. visit  a Participatory Process admin 
3. Add a new category from sidebar 
4. Add a new meeting and assign it to category newly created
5. Visit again the category page and see you cannot delete it 
6. Visit meetings and delete
7. Repeat step 5
8. Apply patch **and restart your server**
9. Visit again the categories and see you can delete your category. 
10. For sanity, repeat 3, 4, 5, 6 then you could jump to step 9 

### :camera: Screenshots
[Screencast from 2023-12-18 18-34-08.webm](https://github.com/decidim/decidim/assets/105683/14560923-3c81-44a1-be99-8e259a407c8a)


:hearts: Thank you!
